akka-stream-tests/src/test/scala/akka/stream/scaladsl/FramingSpec.scala

```diff
@@ -93,6 +93,12 @@ class FramingSpec extends AkkaSpec {
           Source.single(ByteString("ab\n")).via(simpleLines("\n", 1)).limit(100).runWith(Sink.seq),
           3.seconds)
       }
+
+      an[FramingException] should be thrownBy {
+        Await.result(
+          Source.single(ByteString("aaa")).via(simpleLines("\n", 2)).limit(100).runWith(Sink.seq),
+          3.seconds)
+      }
```

Comment:
Sorry again... My earlier comment was wrong? Looking at the below existing test case, the delimiter bytes are not taken into account when checking maximumBytes.

```
      // The buffer will contain more than 1 bytes, but the individual frames are less
      Await.result(
        Source.single(ByteString("a\nb\nc\nd\n")).via(simpleLines("\n", 1)).limit(100).runWith(Sink.seq),
        3.seconds) should ===(List("a", "b", "c", "d"))
```